### PR TITLE
Rspec Checks for Tic-Tac-Toe game

### PR DESCRIPTION
**Pull request by sergioobolevich**
We created some rspec checks for our Tic-Tac-Toe game public functions with all possible cases:
- #make_move
- #check_if_win?
- #check_if_move_done

### CODE REVIEW

File: spec/methods_spec.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  let(:player) { Player.new('Sergio', 'X') }
+  describe '#make_move' do
+    it 'return true if user move is correct' do
+      expect(new_game.make_move(player.team, 4)).to eql TRUE
+    end
+    it 'return false if the move is not a number' do
+      expect(new_game.make_move(player.team, 'string')).to eql FALSE
+    end
+    it 'return false if the move is not a number between 1-9' do
+      expect(new_game.make_move(player.team, 12)).to eql FALSE
```
Comment (kitpao) [inline spec/methods_spec.rb]: - [x] **OPTIONAL** In all your tests, you have these warnings about deprecated constant, you can fix it by replacing `TRUE` with `true` and `FALSE` with `false`
image

File: spec/methods_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    end
+  end
+
+  describe '#check_if_win?' do
+    it 'returns false when there is no winner' do
+      expect(new_game.check_if_win?).to eql FALSE
+    end
+  end
```
Comment (kitpao) [inline spec/methods_spec.rb]: - [x] **MANDATORY** Please add a test case for when there IS a winner
image
The thing there is that you cannot have a winner and a loser at the same time.

File: spec/methods_spec.rb [hunk 3]
```diff
@@ -0,0 +1,33 @@
+require './lib/logic'
+
+describe Game do
+  let(:new_game) { Game.new }
+  let(:player) { Player.new('Sergio', 'X') }
```
Comment (kitpao) [inline spec/methods_spec.rb:5]: - Good use of `let` for variable declaration 💯

**Review by kitpao — CHANGES_REQUESTED** [review]
# 👋 Hello @chubaquelo and @Kingobaino1 

🌟 Nice work with your tests, all your tests are passing and almost complete 😄 Please check below to check the missing requirements so that your project can be approved 📝 

**_Note:_** All the **[OPTIONAL]** are **not required** in order to approve your project but are personal suggestions to improve your coding experience

- [x] **MANDATORY** Now that you have your tests, please add some instructions to your Readme so a user can run your tests.
- [x] **OPTIONAL** Related to the previous point, you can use backquotes to give code formatting `like this` 
  ```
  `like this` 
  ```
- [x] **MANDATORY** I don't know what happened, but your main.rb file is not running with `bin/main.rb` command, kindly fix it. **Hint:** you can use the `chmod 755 bin/main.rb` command on Linux to add the corresponding permissions
image

- [x] **MANDATORY** Your linters are not displaying in your pull request, please check the configuration until you get a message similar to the one in the image. **Hint:** For next time try to add the linters immediately after you create the branch to avoid these issues, for now, I would recommend you to try adding the .rubocop.yml file from [this link.](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)
image
- [x] Don't forget inline feedback below 💡 

### ♻️ Status: Your project requires some additional changes ♻️
➡️ Please request another review once you are ready 📝 
___
##### 👉 If you have any doubts you can contact me on slack as @kitpao
##### 🌟  If you have feedback about my reviews I would appreciate it if you could leave a recommendation in LinkedIn 💌 or let me know in Slack if you see improvements to be done 😄  

🔗 __[Github](https://github.com/kitpao) | [Twitter](https://twitter.com/Kitpao1) | [LinkedIn](https://www.linkedin.com/in/kitzia-paola-vidal/) | [Medium](https://medium.com/@kitpao91)__

File: spec/methods_spec.rb (hunk 2 above)
Comment (Kingobaino1) [reply spec/methods_spec.rb]: > * [x]  **MANDATORY** Please add a test case for when there IS a winner
>   image
- The thing there is that you cannot have a winner and a loser at the same time.
- When there is a winner and when there is no winner are mutually exclusive.

**Review by javitocor — APPROVED** [review]
# STATUS APPROVED ⭐  🥇 

Congratulations team! 🥳 
You have met all the requirements 👏  and your project is approved 👍 
Remember to fill up the completion form 📑  and move into the next project ⏩ 

Good luck and keep the good job up! 😃

**Pull request merged**